### ISSUE TITLE
Not able to replicate results mentioned 

### ISSUE THREAD

**Issue by NiranthS** [clvrai/spirl#7]
Hello @kpertsch and @youngwoon 
I tried training the skill prior module using the block-stacking dataset in the given configuration. The overloss went from 15 to 11. When I use this pretrained model as a skill prior module for SAC, I am not able to get proper results.
I am not sure, but I think the skill prior module did not converge properly on the training dataset.
Can you please provide me with pretrained models/events files you got during training. I would be able to interpret which part of the model is not performing well. Also, do you have any other suggestions on replicating the results mentioned in the paper.
Thank You

**Comment by kpertsch** [comment]
Hi Niranth,

Sorry for the late reply. I just pushed an update for the block stacking environment that uses the improved closed-loop action decoder with image observations -- I think you should use this as the default model for this environment. In my experiments it lead to improved performance over the original SPiRL model even on a more challenging, sparse reward version of the block stacking task.

Can you please re-train the closed-loop SPiRL model for block stacking with the example commands [here](https://github.com/clvrai/spirl/blob/master/spirl/configs/skill_prior_learning/block_stacking/hierarchical_cl/README.md). Below I attached my training curves for this model for your reference.

![image](https://user-images.githubusercontent.com/15955682/116468051-b0be5800-a870-11eb-9b32-f367c4b920ee.png)

**Comment by NiranthS** [comment]
Hello Karl Pertsch
Thanks a lot for the info. I will try it out and let you know if I face any further problems.
